street_lights/js/swiatla.js

```diff
@@ -0,0 +1,38 @@
+  var swiatloRed = document.getElementsByClassName('light red');
```

Comment:
Super! 

Postaraj się teraz ogarnąć to na tablicy.

var lights = document.getElementsByClassName('light')

var redLight = lights[0]
var yellowLight = lights[1]
var greenLight = lights[2]